### PR TITLE
Basic Launch file

### PR DESCRIPTION
**Pull request by Jailander**
as discussed in #361 this PR adds the argument `launch_manager` to topological navigation launch files, this allows the possibility of not launching the map manager, which is useful in multi-robot systems.
Also, this PR creates a basic launch file that doesn't include travel time prediction services or policy execution for simple usage cases.

### CODE REVIEW

File: topological_navigation/launch/topological_navigation_basic.launch [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+	<arg name="topological_navigation_retries" default="3"/>
+	<arg name="machine" default="localhost" />
+	<arg name="user" default="" />
+        <arg name="launch_manager" default="true"/>
+	<arg name="move_base_reconf_service" default="DWAPlannerROS" />
+
+	<machine name="$(arg machine)" address="$(arg machine)" env-loader="$(optenv ROS_ENV_LOADER )" user="$(arg user)" default="true"/>
+
+ 	<node pkg="monitored_navigation" type="monitored_nav.py" name="monitored_nav" output="screen" args="$(arg mon_nav_config_file)">
+		<param name="wait_reset_bumper_duration" value="$(arg wait_reset_bumper_duration)"/>
```
Comment (marc-hanheide) [inline topological_navigation/launch/topological_navigation_basic.launch:18]: I think this basic topo nav shouldn't have these robot specific recoveries

File: topological_navigation/launch/topological_navigation_basic.launch [hunk 2]
```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" ?>
+
+<launch>
+	<!-- declare arg to be passed in -->
+	<arg name="map"/>
+	<arg name="mon_nav_config_file"  default="" />
+	<arg name="max_bumper_recoveries" default=".inf"/>
+	<arg name="wait_reset_bumper_duration" default="0.0"/>
```
Comment (marc-hanheide) [inline topological_navigation/launch/topological_navigation_basic.launch:8]: see my other comment. bumper is robot-specific.

**Review by marc-hanheide — CHANGES_REQUESTED** [review]

File: topological_navigation/launch/topological_navigation_basic.launch (hunk 1 above)
Comment (Jailander) [reply topological_navigation/launch/topological_navigation_basic.launch:18]: true, I will take a look at how to remove these requirements

File: topological_navigation/launch/topological_navigation_basic.launch (hunk 2 above)
Comment (Jailander) [reply topological_navigation/launch/topological_navigation_basic.launch]: (y)

**Review by marc-hanheide — COMMENTED** [review]
Any progress?

**Pull request closed without merging**